pam_turnstile allows turnstile to integrate with gdm

See also: https://github.com/chimera-linux/cports/blob/master/main/gdm/patches/pam.patch

--- a/data/pam-arch/gdm-launch-environment.pam
+++ b/data/pam-arch/gdm-launch-environment.pam
@@ -14,6 +14,7 @@
 session    required                    pam_succeed_if.so    audit quiet_success user in gdm:gnome-initial-setup
 session    optional                    pam_permit.so
 -session   optional                    pam_systemd.so
+-session   optional                    pam_turnstile.so
 -session   optional                    pam_elogind.so
 session    required                    pam_env.so
